### PR TITLE
#388 migrate org.jpeek.web to junit 5

### PR DESCRIPTION
**Pull request by HDouss**
Resolve puzzle in #388

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#391](https://github.com/yegor256/jpeek/issues/391) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/4266805b-0e13-4e79-b86f-f4fec82775b9); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by HDouss** [comment]
@fanifieiev ping

File: src/test/java/org/jpeek/web/ReportsTest.java [hunk 1]
```diff
@@ -46,20 +46,21 @@
...
+            Assumptions.assumeTrue(false);
         }
     }
 
     @Test
     public void rendersOneReport() throws Exception {
-        final BiFunc<String, String, Func<String, Response>> reports =
-            new Reports(Files.createTempDirectory("x"));
+        final BiFunc<String, String, Func<String, Response>> reports = new Reports(
+            Files.createTempDirectory("x")
```
Comment (fanifieiev) [inline src/test/java/org/jpeek/web/ReportsTest.java]: @HDouss Why not using `org.junit.jupiter.api.io.TempDir` ?

Comment (HDouss) [reply src/test/java/org/jpeek/web/ReportsTest.java]: @fanifieiev Fixed it! I did not do it as it is not JUnit 4 api, so I did not take that as migration. We have too many of these, and I submitted a separate issue for this: #394

Comment (fanifieiev) [reply src/test/java/org/jpeek/web/ReportsTest.java]: @HDouss Ok

**Comment by fanifieiev** [comment]
@paulodamaso good to merge

**Comment by HDouss** [comment]
@paulodamaso ping

**Comment by paulodamaso** [comment]
@HDouss please solve conflicts

**Comment by HDouss** [comment]
@paulodamaso Done!

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by HDouss** [comment]
@paulodamaso Are you sure rultor is taking its config from master, and not from this branch? If you are sure your modifications in 801486c are the solution, I can remerge master.

**Comment by HDouss** [comment]
@paulodamaso merged again. Could you please try to merge again?

**Comment by HDouss** [comment]
@paulodamaso ping

**Comment by 0crat** [comment]
@fanifieiev[/z](https://www.0crat.com/u/fanifieiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by HDouss** [comment]
@paulodamaso Could we merge this also?

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@fanifieiev please make sure you will try to find at least 3 issues during next CR, thanks

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by fanifieiev** [comment]
> @fanifieiev please make sure you will try to find at least 3 issues during next CR, thanks

@sereshqua Got it! Sorry for making that happen again and again.

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**